packages/shortcuts-extension/src/components/ShortcutItem.tsx

```diff
@@ -469,7 +605,13 @@ export class ShortcutItem extends React.Component<
     } else {
       return (
         <div
+          title={this.props.shortcut.commandName}
```

Comment:
```suggestion
          title={this.props.shortcut.label}
```


`commandName` is something like "my-package:my-command". Label is the thing which is user-readable.